File: programmers/java/lev0/모음 제거.java
import java.util.stream.Collectors;
import java.util.stream.Stream;
class Solution {
    public String solution(String my_string) {
        // return my_string.chars().filter(el->el !='a'&& el !='i'&& el !='u'&& el !='e'&& el !='o').mapToObj(Character::toString).collect(Collectors.joining());
        return my_string.replaceAll("[aeiouAEIOU]", "");

    }
}

File: programmers/js/lev0/모음 제거.js
function solution(my_string) {
  // return my_string.split("").map(el => el!='a' && el!='i' && el!='u' && el!='e' && el!='o' ? el:"").join("")

  // 정규표현식 /[]/g
  return string.replace(/[aeiou]/g, "");
}
